packages/cli/index.js

```diff
@@ -230,7 +230,8 @@ function assertTd(tds, type, tdsToMerge, manualAssertions, doneEventEmitter) {
         if (tds.length > 0) {
             tdAssertions(tds, fileLoader, logFunc, manualAssertions, doneEventEmitter).then( results => {
                 if (type === "file") {
-                    outReport(results, "assertionsTest_", input)
+                    const id = input.split('/').pop()
```

Comment:
what is happening here? Can you comment

Reply:
Basically what is happening is the outReport function takes the `input (-i)` parameter from the CLI and uses that as part of the `id` in the `output` path. The problem happens when you specify a path for both the `input (-i)` and the `output (-o)`.  When you do this, both get concatenated together and produce an error. I made sure that from the input path we basically take only the name of the file as for the `id` in the output path

Reply:
I see, that is good. Could you add this as a comment

Reply:
I found where the problem was, it wasn't in this line, so I reverted that line to its former state

Reply:
I found where the problem truly is. I fixed that and now it should work